support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -142,4 +142,24 @@ export const tests: Tests = {
 		targetPage: pageUrlRegexes.contributions.allLandingPagesAndThankyouPages,
 		excludeCountriesSubjectToContributionsOnlyAmounts: true,
 	},
+	abandonedBasket: {
+		variants: [
+			{
+				id: 'control',
+			},
+			{
+				id: 'variant',
+			},
+		],
+		audiences: {
+			ALL: {
+				offset: 0,
+				size: 1,
+			},
+		},
+		isActive: true,
+		referrerControlled: false, // ab-test name not needed to be in paramURL
+		seed: 1,
+		excludeCountriesSubjectToContributionsOnlyAmounts: true,
+	},
```

Comment:
I've set it as `contributions.allLandingPagesAndThankyouPages` - we're only interested in the three tier checkout for this so that seems to be as specific as we can go

Reply:
Yep this is the best target to use 👍 .

It could be worth catching up with @jamesgorrie regarding the current generic checkout A/B test and whether this impacts the abandoned basket work? In short we have an A/B test running at the moment testing a new "generic" checkout for users coming through Tier 1.